rhel.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.1-409
+
+ARG ARCH=amd64
```

Comment:
we can automatically detect the arch using uname (so it works either with Docker build or Docker buildx)